storey/table.py

```diff
@@ -1027,6 +1033,9 @@ def reset(self, value=None):
         else:
             self.value = float(value)
 
+    def aggregate_lua_script(self, vl1, vl2):
+        return 'type({}) == "number" and math.min({},{}) or {}'.format(vl1, vl1, vl2, vl2)
```

Comment:
Any reason not to use modern f notation? i.e.:
```
return f'type({vl1}) == "number" and math.min({vl1},{vl2}) or {vl2}'
```
The same applies to other uses of `format()`.

Reply:
Agree

Reply:
fixed